src/js/main.js

```diff
@@ -597,6 +599,14 @@
         document.addEventListener("keydown", e => {
             keysDown[e.which] = true;
 
+            // Start the level when enter is pressed.
+            if (!gameStarted) {
```

Comment:
As a minor note, perhaps there should also be condition `e.which === KEY_ENTER` here.